docs/api/notification.md

```diff
@@ -29,9 +29,9 @@ Returns `Boolean` - Whether or not desktop notifications are supported on the cu
 ### `new Notification([options])`
 
 * `options` Object (optional)
-  * `title` String - A title for the notification, which will be shown at the top of the notification window when it is shown.
+  * `title` String (optional) - A title for the notification, which will be shown at the top of the notification window when it is shown.
```

Comment:
Technically `title` isn't optional.  Doesn't `toastXml` just completely replace all the other options in this structure.  Maybe this would make more sense if it was `new Notification(toastXml)`.  I.e. `options Object | String (optional)`

Reply:
yeah, its optional because when you provide Toast XML you don't need it. I thought about `options Object | String (optional)` Not sure how to implement on the API though.

Reply:
The notification will work without title and body. It actually did always because of a bug in the XML creation and Windows handling an empty notification gracefully. So if you provide an empty object nothing will blow up.